src/components/Events.tsx

```diff
@@ -0,0 +1,48 @@
+import type { FC } from "react";
+
+export const mockEvents = [
+  {
+    id: 1,
+    name: "Vendy",
+    sport: "Basketball",
+  },
+  {
+    id: 2,
+    name: "Lukas",
+    sport: "Football",
+  },
+  {
+    id: 3,
+    name: "Honza",
+    sport: "Tennis",
+  },
+  {
+    id: 4,
+    name: "Vendy",
+    sport: "Box",
+  },
+  {
+    id: 5,
+    name: "Lukas",
+    sport: "Skydiving",
+  },
+  {
+    id: 6,
+    name: "Johana",
+    sport: "Jogging",
+  },
+];
+
+export const Events: FC = () => {
+  return (
+    <>
+      {mockEvents.map((event) => (
+        <article key="id">
```

Comment:
The key should be some unique value for each of the items, which is also stable (so Math.random() won't cut it).

Check - https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

In our example the ideal would be doing
```suggestion
        <article key={event.id}>
```

Reply:
Damn, that is what I meant! 😅